packages/app/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "app",
+  "private": true,
+  "version": "0.0.0",
+  "description": "Lambdas to maintain and forward cloudwatch logs",
+  "scripts": {
+    "build": "tsc",
+    "test": "jest"
+  },
+  "devDependencies": {
+    "@types/aws-lambda": "^8.10.20",
+    "aws-sdk": "^2.1159.0"
```

Comment:
Only the app uses these dependencies, so are included in `packages/app/package.json` _not_ the root `package.json`.